### PR TITLE
feat: add support for import maps

### PR DESCRIPTION
**Pull request by lucacasonato**

### CODE REVIEW

File: action/index.js [hunk 1]
```diff
@@ -47,6 +48,20 @@ async function main() {
   }
   core.info(`Entrypoint: ${url.href}`);
 
+  let importMapUrl = null;
+  if (importMap) {
+    importMapUrl = await parseEntrypoint(importMap, cwd, "import map");
+    if (importMapUrl.protocol === "file:") {
+      const path = fromFileUrl(importMapUrl);
+      if (!path.startsWith(cwd)) {
+        throw "Import map must be in the current working directory.";
```
Comment (satyarohith) [inline action/index.js:57]:
```suggestion
        throw "Import map must be in the current working directory or specified root directory.";
```

**Review by satyarohith — APPROVED** [review]
LGTM.

File: examples/hello.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import { serve } from "https://deno.land/std@0.114.0/http/server.ts";
+import { serve } from "std/http/server.ts";
```
Comment (ry) [inline examples/hello.ts:1]: Is this exercised in the tests?

Comment (lucacasonato) [reply examples/hello.ts:1]: Yes, see `.github/workflows/test.yml`

**Review by ry — APPROVED** [review]
lgtm

**Pull request merged**